src/addRemove.js

```diff
@@ -0,0 +1,83 @@
+import TrashImg from './delete.svg';
+import MoreImg from './more.svg';
+
+const addTask = (tasks) => {
+  const str = document.getElementById('description').value;
+  const firstLetter = str.charAt(0).toUpperCase();
+  str.replace(str.charAt(0), firstLetter);
+  const description = str;
+  const completed = false;
+  const date = new Date();
+  const id = date.getMilliseconds();
+
+  if (!tasks) {
+    tasks = [];
+  }
+
+  const index = tasks.length + 1;
+
+  if (tasks && description !== '') {
+    const task = {
+      description,
+      completed,
+      index,
+      id,
+    };
+    tasks.push(task);
+    tasks.sort((taskA, taskB) => {
+      const indexA = taskA.position;
+      const indexB = taskB.position;
+      if (indexA < indexB) {
+        return -1;
+      }
+      if (indexA > indexB) {
+        return 1;
+      }
+      return 0;
+    });
+    window.update(tasks);
+  }
+};
+
+const removeTask = (data, tasks) => {
+  const str = data.replace('div', '');
+  const newTasks = [];
+  tasks.forEach((task) => {
+    if (task.index !== parseInt(str, 10)) {
+      newTasks.push(task);
+    }
+  });
+  window.update(newTasks);
+};
```

Comment:
- [ ] After removing elements from the task array, please make sure to update the indexes of all the remaining numbers in order to have a sequence of consecutive indexes in the remaining array.